dotnetcore-acquisition-extension/package.json

```diff
@@ -1,16 +1,16 @@
 {
-	"name": "dotnetcore-acquisition",
+	"name": "vscode-dotnet-runtime",
```

Comment:
Any reason you're not also changing the folder name `dotnetcore-acquisition-extension` -> `vscode-dotnet-runtime-extension`

Reply:
Not in particular, I figure that it's not exposed externally so might as well keep it the same. I can update it if you think that's more clear.

Reply:
Would probably make it more clear to have the repo consistent but not a big deal either 😄

Reply:
Latest commit has the folder names updated 👍